app/js/lib/foxbox/sequential-timer.js

```diff
@@ -0,0 +1,94 @@
+const p = Object.freeze({
```

Comment:
I don't really have an opinion, pick the one you like best.
For the moment only `eventDispatcher` will go in there, right?